supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -590,6 +590,19 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*:
 
+[discrete]
+[[softirq]]
+==== image:images/yes.png[yes] softirq (noun)
+*Description*: A _software interrupt request (softirq)_ is a deferrable kernel routine that performs the required actions in response to an interrupt. For example, softirqs clear the receive ring buffer after a network adapter receives a packet.
+
+Capitalize the initial "S" only at the beginning of a sentence.
```

Comment:
We don't usually state specifically when a glossary term is normally uncapitalized, unless it should be uncapitalized even at the beginning of a sentence. Recommend deleting this line.

Reply:
@mportman12 I added this sentence for consistency with other glossary entries that require lowercase, but which are OK to capitalize at the beginning of the sentence. For example, [hertz](https://redhat-documentation.github.io/supplementary-style-guide/#hertz), [host group](https://redhat-documentation.github.io/supplementary-style-guide/#host-group), [hypervisor](https://redhat-documentation.github.io/supplementary-style-guide/#hypervisor), and [jsvc](https://redhat-documentation.github.io/supplementary-style-guide/#jsvc).

The IBM Style Guide also spells out this exception. For example:

```
✓ ebook adjective, noun
Write as shown: one word, unhyphenated. If this term occurs at the beginning of a sentence or title, use Ebook.

✓ email adjective, noun, verb
Write as shown: one word, unhyphenated. If this term occurs at the beginning of a sentence or title, use Email.
```

Should I remove it anyway?


> ...unless it should be uncapitalized even at the beginning of a sentence

Do our style guides allow that we begin sentences with lower case? I always avoided this, for example:

"The firewalld service uses the concept of zones..."

instead of

"firewalld uses the concept of zones..."

Reply:
I agree. Better to restructure a sentence so that it starts with a caps letter. 

![Uploading Screenshot from 2023-04-25 10-32-39.png…]()